### PR TITLE
Added Postgres

### PR DESCRIPTION
**Pull request by mssiegel**
See #5 

Added setup for Postgres to app.py with tables for Customers and Chefs

Updated Readme to include instructions for setting up Postgres

### CODE REVIEW

**Review by giftofgrub — COMMENTED** [review]
Before the commands to initalize Postgres, I think it should have instructions to make sure that the virtual environment is activated "pipenv shell" so that when you call "from app import db" it does not error out.

File: server/app.py [hunk 1]
```diff
@@ -9,3 +11,70 @@
...
+DB_URL = 'postgresql+psycopg2://{user}:{pw}@{url}/{db}'.format(user=user,pw=pw,url=url,db=db_name)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = DB_URL
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+ma = Marshmallow(app)
+
+# Basic User Model from which Customers and Chefs inherit from
+class User(db.Model):
```
Comment (skassam21) [inline server/app.py]: Is there a way we can get these classes to exist in a directory like (server/models/user.py)

File: server/app.py [hunk 2]
```diff
@@ -9,3 +11,70 @@
...
+
+customer_schema = ChefSchema()
+customers_schema = ChefSchema(many=True)
+
+
+# Routes for Customers
+
+# Create a Customer
+@app.route('/customer', methods=['POST'])
+def add_customer():
```
Comment (skassam21) [inline server/app.py]: Also, we'd probably want these to be in api folder

Comment (skassam21) [reply server/app.py]: Did you also want to look into Flask-RESTful

File: README.md [hunk 3]
```diff
@@ -2,7 +2,22 @@
 
 ## Starting the server:
 
-
 1. Open a terminal and go to the server folder. Make sure you have **pipenv** installed (`pip install pipenv`)
 2. Install the dependencies with `pipenv install`. This also createa a virtual environment, if there isn't one already
 3. Activate the virtual environment and start the app with `pipenv run flask run`
```
Comment (skassam21) [inline README.md]: Get into the shell with this: `pipenv shell`
Then you can do whatever you want (like flask run)

File: server/app.py (hunk 1 above)
Comment (giftofgrub) [reply server/app.py]: we can store the database variables in the .env then access it by 
```
import os
os.environ["DB_URL"]
```

**Comment by mssiegel** [comment]
I converted routes to Flask-RESTful and moved them into api folder. I also moved the models into their own folder.

File: server/app.py [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
+    app = Flask(__name__)
+
+    app.register_blueprint(home_handler)
+    app.register_blueprint(ping_handler)
+
+    # Database variables
+    user = 'postgres'
+    pw = '123456'
+    url = 'localhost:5432'
+    db_name = 'team-bluejay'
```
Comment (skassam21) [inline server/app.py:23]: these database variables should be coming from a config file, and that config file should be referencing environment variables

**Review by skassam21 — APPROVED** [review]

**Pull request merged**